packages/ddex/ingester/parser/ern38x.go

```diff
@@ -337,7 +360,6 @@ func processReleaseNode(rNode *xmlquery.Node, soundRecordings *[]SoundRecording,
 				DDEXReleaseIDs:      *ddexReleaseIDs,
 				Genre:               genre,
 				IsAlbum:             true,
-				IsPrivate:           false, // TODO: Use DealList to determine this. Same with releaseDate because I think the XML element it's reading is deprecated
```

Comment:
I didn't see anything in the deals I looked at wrt this. what is `isPrivate`?

Reply:
glad you deleted this comment. i was thinking that DealList controls everything, but i guess that's not the case for IsPrivate because DDEX probably doesn't support private tracks like what Audius has